cmake/Platform/Linux.cmake

```diff
@@ -25,9 +25,15 @@ endif()
 
 message(STATUS "CMAKE_DATA_PATH: ${CMAKE_DATA_PATH}.  Change this by invoking cmake with -DCMAKE_DATA_PATH=<SOME_DIRECTORY>")
 
-# Quotes need to be a part of the definition or the compiler won't understand
-add_definitions(-DLINUX_DATA_DIR="${CMAKE_DATA_PATH}")
-
+# The executable will look for game assets in LINUX_DATA_DIR, hardcoded.
+# The default here is fine, but we are overridable in case a packager needs to move these after the fact.
+if (NOT LINUX_DATA_DIR)
+	# Quotes need to be a part of the definition or the compiler won't understand
+	add_definitions(-DLINUX_DATA_DIR="${CMAKE_DATA_PATH}")
+else()
+	# Yes, this is tautological. No, it does not work without it.
```

Comment:
not proud of snarky comments, but this legit had me pulling my hair out. Happy to rephrase. Edits are on if you have a better solution.

Reply:
This was a line I had to break for my AppImage build, too.  I think the proper solution may be to remove it completely and modify the `getInstalledDataDir()` function in `zap/SystemFunctions.cpp` to be smarter in some way and use relative paths.

Reply:
I'd buy this needs a cleanup. How's your appimage build? Happy to take a whack at cleaning up if you don't think that's likely to merge for a while.

Reply:
If you want to take a stab at cleaning it up, go for it.  The main issue is that the code base need to know where to find the game resources, and since Linux distros can put them wherever, that LINUX_DATA_DIR flag was used. It should probably not be set as default though...

Reply:
I didn't take a run at this for now, but might do so if I add flatpak support (if that's a thing you guys would like, i'm interested academically but not likely to spend the time to get it working if it's not something youd like to have).